Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -281,25 +280,25 @@ private fun JetsnackBottomNavItemLayout(
     @FloatRange(from = 0.0, to = 1.0) animationProgress: Float
 ) {
     Layout(
-        children = {
-            Box(Modifier.layoutId("icon"), children = icon)
+        content = {
+            Box(Modifier.layoutId("icon"), content = icon)
             val scale = lerp(0.6f, 1f, animationProgress)
             Box(
                 modifier = Modifier
                     .layoutId("text")
                     .padding(start = TextIconSpacing)
-                    .drawLayer(
+                    .graphicsLayer(
```

Comment:
Nit: use the new lamdba for `graphicLayer`. It means that only the lambda will be restarted on anim changes, rather than the whole function.

Reply:
Done